bin/test_travis.sh

```diff
@@ -99,7 +99,7 @@ ctest --output-on-failure
 # Python
 if [[ "${WITH_PYTHON}" == "yes" ]] && [[ "${WITH_SAGE}" != "yes" ]]; then
     cd symengine/python
-    nosetests -v
+    py.test -v
```

Comment:
I thought we had that in sympy. Don't worry about it now --- let's see if we can get this PR working with older version of Cython. Then we can worry about this. I can help with this as well.